### ISSUE TITLE
forward port new picard functionality--query-sorted mark duplicates

### ISSUE THREAD

**Issue by yfarjoun** [broadinstitute/gatk#1636]
see: https://github.com/broadinstitute/picard/pull/491

**Comment by akiezun** [comment]
This is blocked by #1210 
(note though that MarkDuplicatesSpark already defaults to query-sorted input because it improves speed by 2x by doing only 1 shuffle).

**Comment by yfarjoun** [comment]
interesting, but the functionality should change then to mark the secondary
and supplementaries and unmapped mates..

**Comment by magicDGS** [comment]
This can be closed as Picard is pull-out as a dependency, no @droazen?

**Comment by droazen** [comment]
Closing as duplicate of https://github.com/broadinstitute/gatk/issues/3705
